prow/gerrit/reporter/reporter.go

```diff
@@ -115,17 +115,16 @@ func (c *Client) ShouldReport(pj *v1.ProwJob) bool {
 		return false
 	}
 
-	// Only report when all jobs of the same type on the same revision finished
-	selector := labels.Set{
-		client.GerritRevision: pj.ObjectMeta.Labels[client.GerritRevision],
-		kube.ProwJobTypeLabel: pj.ObjectMeta.Labels[kube.ProwJobTypeLabel],
+	// Don't wait for report aggregation if not voting on any label
+	if pj.ObjectMeta.Labels[client.GerritReportLabel] == "" {
+		return true
 	}
 
-	if pj.ObjectMeta.Labels[client.GerritReportLabel] == "" {
-		// Shouldn't happen, adapter should already have defaulted to Code-Review
```

Comment:
We need to get rid of the defaulting: https://github.com/kubernetes/test-infra/blob/72789fc5254d7cc3bca60479acffcbc791ab3c7c/prow/gerrit/adapter/adapter.go#L286-L288

Reply:
Looks like we were defaulting twice... https://github.com/kubernetes/test-infra/blob/72789fc5254d7cc3bca60479acffcbc791ab3c7c/prow/gerrit/reporter/reporter.go#L199-L202

Reply:
Nice catch. I'll still prefer unspecified to default to "Code-Review" for backwards compatibility.